velox/core/PlanNode.h

```diff
@@ -566,7 +571,8 @@ class GroupIdNode : public PlanNode {
   GroupIdNode(
       PlanNodeId id,
       std::vector<std::vector<FieldAccessTypedExprPtr>> groupingSets,
-      std::map<std::string, FieldAccessTypedExprPtr> outputGroupingKeyNames,
+      std::map<int, std::shared_ptr<OutputGroupingKeyInfo>>
```

Comment:
OutputGroupingKeyInfo is a simple struct. Is there any particular reason to use it with a shared_ptr?